Sources/Growth/URP/UserReferralProgram.swift

```diff
@@ -7,6 +7,8 @@ import Shared
 import Preferences
 import WebKit
 import os.log
+import iAd
```

Comment:
`iAD` import does not seem needed

Reply:
Fix at https://github.com/brave/brave-ios/pull/8194/commits/13613149ef5a8e66220586eeef7edd1d2bef696b